.travis.yml

```diff
@@ -58,6 +58,16 @@ jobs:
           skip_cleanup: true
           on:
             tags: true
+    - stage: Publish latest Docker Image on Daily Build
+      before_deploy:
+        - export TRAVIS_TAG=$(git describe --tags)
+      script:
+        - echo "Publish latest Docker Images to DockerHub"
+      deploy:
+        - provider: script
+          script: make TAG=${TRAVIS_TAG} RELEASE_VER=latest release
```

Comment:
We need always generate the images with `latest` tag, since it's what we used in the all-in-one yaml.

Reply:
got it..updated